extensions/wikia/AssetsManager/config.php

```diff
@@ -384,7 +384,7 @@
 	'type' => AssetsManager::TYPE_JS,
 	'assets' => array(
 		'//extensions/wikia/AdEngine/js/Exitstitial.js',
-		'//extensions/wikia/UserLogin/js/UserSignupMarketingOptIn.js',
+		'//extensions/wikia/UserLogin/js/MarketingOptIn.js',
```

Comment:
Since you're renaming this file, what do you think about also changing the directory from js to scripts to follow the pattern found elsewhere?